python/GangaLHCb/__init__.py

```diff
@@ -46,27 +46,8 @@
     configLHCb.addOption('SplitByFilesBackend', 'OfflineGangaDiracSplitter',
                      'Possible SplitByFiles backend algorithms to use to split jobs into subjobs,\
                       options are: GangaDiracSplitter, OfflineGangaDiracSplitter, splitInputDataBySize and splitInputData')
-
-
-def _guess_version(name):
-    if 'GANGASYSROOT' in os.environ.keys():
-        gangasys = os.environ['GANGASYSROOT']
-    else:
-        raise OptionValueError("Can't guess %s version if GANGASYSROOT is not defined" % name)
-    tmp = tempfile.NamedTemporaryFile(suffix='.txt')
-    cmd = 'cd %s && cmt show projects > %s' % (gangasys, tmp.name)
-    rc = subprocess.Popen([cmd], shell=True).wait()
-    if rc != 0:
-        msg = "Fail to get list of projects that Ganga depends on"
-        raise OptionValueError(msg)
-    p = re.compile(r'^\s*%s\s+%s_(\S+)\s+' % (name, name))
-    for line in tmp:
-        m = p.match(line)
-        if m:
-            version = m.group(1)
-            return version
-    msg = 'Failed to identify %s version that Ganga depends on' % name
-    raise OptionValueError(msg)
+    defaultLHCbDirac = 'v8r2p36'
+    configLHCb.addOption('LHCbDiracVersion',defaultLHCbDirac,"set LHCbDirac version")
```

Comment:
Minor issue: There should be spaces after the commas in the argument list
Even more minor issue: There's an inconsistent use of types of quotation marks on this line. Perhaps just use `'` here?